docker-compose-local.yml

```diff
@@ -15,11 +15,19 @@ services:
     container_name: frontend
     build: frontend
     command: yarn serve
+    environment:
+      - VITE_VUE_APP_MATOMOID=${VITE_VUE_APP_MATOMOID}
+      - VITE_VUE_APP_HOTJAR=${VITE_VUE_APP_HOTJAR}
     volumes:
       - ./frontend/src:/project/src
       - ./frontend/public:/project/public
       - ./frontend/.eslintrc.js:/project/.eslintrc.js
       - ./frontend/.prettierrc.json:/project/.prettierrc.json
+      - ./frontend/vite.config.js:/project/vite.config.js
+      - ./frontend/package.json:/project/package.json
```

Comment:
I'm not sure what the dev flow is for everyone but I imagine what we had to do before when installing any new packages was to modify the `package.json` file and then run `build-stack` to get the changes. I thought it was more convenient, especially during this PR, to be able to either modify the `package.json` and then run `ma-exec frontend yarn`, or run `ma-exec frontend yarn add PACKAGE_NAME` and to have the `package.json` file changes reflected to outside the docker container.

Since this is not specifically required and something we had discussed about prior, I'm happy to take this out. I don't think it hurts to leave it in either though. Please let me know!

Reply:
I see. Another way to do this would be to run `docker cp` after the new package is added in the container. Personally I prefer the way it was before, but I don't see any problem in keeping things as they are now.